torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20416,7 +20416,8 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
     OpInfo(
         "norm",
         sample_inputs_func=sample_inputs_norm,
-        dtypes=floating_and_complex_types_and(torch.float16, torch.bfloat16),
+        dtypes=floating_and_complex_types_and(torch.float16, torch.bfloat16, torch.chalf),
+        dtypesIfCUDA=floating_and_complex_types_and(torch.float16, torch.bfloat16),
```

Comment:
Can we just add chalf support to CUDA too while we are at it? We already support complex types on GPU so should be an easy enough change. Just add kComplexHalf to the cuda operator as well.

Reply:
No, issue [#132634](https://github.com/pytorch/pytorch/issues/132634) also exists on CUDA, there is the same bug in GPU kernel, it runs in here: https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/cuda/ReduceNormKernel.cu#L35